src/uproot_browser/plot.py

```diff
@@ -55,6 +55,8 @@ def plot_branch(tree: uproot.TBranch) -> None:
     array = tree.array()
     values = ak.flatten(array) if array.ndim > 1 else array
     finite = values[np.isfinite(values)]
+    if len(finite) < 1:
+        raise ValueError(f"Branch {tree.name} is empty.")
```

Comment:
I opted to just abuse `ValueError` here instead of making custom exceptions, but if that is preferable I'm happy to do so (we make them quite often in `pyhf`).

Reply:
Using `ValueError` seems fine to me!

Reply:
Is there any chance a ValueError could be raised for another reason? This will claim all value errors are EMPTY. A custom exception is safer.